packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -20,14 +21,16 @@ export default function useTabNav() {
 
 	const { hasMultiSelection, getSelectedBlockClientId, getBlockCount } =
 		useSelect( blockEditorStore );
-	const { setNavigationMode, setLastFocus } = useDispatch( blockEditorStore );
+	const { setNavigationMode, setLastFocus } = unlock(
+		useDispatch( blockEditorStore )
+	);
 	const isNavigationMode = useSelect(
 		( select ) => select( blockEditorStore ).isNavigationMode(),
 		[]
 	);
 
 	const lastFocus = useSelect(
```

Comment:
I think in this file as well, the lastFocus ref is only used in callbacks so there's a micro-optimization that is possible by using `const { getLastFocus } = unlock( useSelect( blockEditorStore ) );` instead and only calling `getLastFocus` within the callbacks.